Sentiment.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+
+# fileName = input("Please enter the text file:")
+
+# Get words from the text file and remove words with less than 3 letters
+def get_sentiment(file_name):
+    negative_words = ["bad", "ugly", "terrible", "awful", "stupid", "mad", "angry", "sad"]
+    file_negative_words = []
+
+    final_words = []
+    test_file = open(file_name, 'r')
+    file_content = test_file.read()
+    split_words = re.split(r'[.,!\s+]\s*', file_content)
+    for word in split_words:
+        if len(word) >= 3:
+            final_words.append(word)
+    total_word_count = len(final_words)
+
+    for word in final_words:
+        if word in negative_words:
```

Comment:
you are expected to count the number of occurrences of the negative word not just it's existence in the word